src/main/java/subway/path/application/PathStrategy.java

```diff
@@ -0,0 +1,15 @@
+package subway.path.application;
+
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+import subway.line.domain.Section;
+import subway.path.application.dto.PathRetrieveResponse;
+import subway.station.domain.Station;
+
+import java.util.List;
+
+public interface PathStrategy {
+    void setEdgeWeight(WeightedMultigraph<Station, DefaultWeightedEdge> graph, Section section, DefaultWeightedEdge edge);
+
+    PathRetrieveResponse findPath(WeightedMultigraph<Station, DefaultWeightedEdge> graph, List<Section> sections, List<Station> stationsInPath, Station sourceStation, Station targetStation);
```

Comment:
인자도 줄일 겸 구현체에서만 외부 라이브러리를 참조하도록 변경하면 좋을 것 같은데 민철님 생각은 어떠신가요?
```suggestion
    PathRetrieveResponse findPath(List<Section> sections, Station sourceStation, Station targetStation);
```